oracle-plugin/src/e2e-test/java/io.cdap.plugin/OracleClient.java

```diff
@@ -52,6 +58,71 @@ public static int countRecord(String table, String schema) throws SQLException,
     }
   }
 
+  public static int validateRecordValues(String schema, String sourceTable, String targetTable)
+    throws SQLException, ClassNotFoundException {
+    String getSourceQuery = "SELECT * FROM " + schema + "." + sourceTable;
+    String getTargetQuery = "SELECT * FROM " + schema + "." + targetTable;
+    try (Connection connect = getOracleConnection()) {
+      connect.setHoldability(ResultSet.HOLD_CURSORS_OVER_COMMIT);
+      Statement statement1 = connect.createStatement(ResultSet.TYPE_SCROLL_SENSITIVE, ResultSet.CONCUR_UPDATABLE,
+                                                     ResultSet.HOLD_CURSORS_OVER_COMMIT);
+      Statement statement2 = connect.createStatement(ResultSet.TYPE_SCROLL_SENSITIVE, ResultSet.CONCUR_UPDATABLE,
+                                                     ResultSet.HOLD_CURSORS_OVER_COMMIT);
+      ResultSet rsSource = statement1.executeQuery(getSourceQuery);
+      ResultSet rsTarget = statement2.executeQuery(getTargetQuery);
+      return resultSetDataCompare(rsSource, rsTarget);
+    }
+  }
+
+  public static int resultSetDataCompare(ResultSet rsSource, ResultSet rsTarget) throws SQLException {
```

Comment:
Add a function comment about what is the function intend to do, and return values implies what sort of comparison.

Reply:
Done